src/core/utilities/index.ts

```diff
@@ -19,3 +19,5 @@
 
 export * from "./errors";
 export * from "./identifiers";
+export * from "./iterator";
+export * from "./random";
```

Comment:
Random doesn't feel like something we should be exporting outside of the sdk. Not sure how to handle that however. It quickly gets confusing of some things in utilities are and some are not.